app/views/questions/index.html.erb

```diff
@@ -3,6 +3,8 @@
   <head>
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.0.js"></script>
```

Comment:
You might want to use an SRI like
<script
  src="https://code.jquery.com/jquery-3.5.0.min.js"
  integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
  crossorigin="anonymous"></script>

Reply:
In terms of best practices, it's generally preferred to have scripts loaded `async`/`defer` and/or near the bottom of `<body>` so the page can visually render first while the scripts load in